src/pages/Landing/index.tsx

```diff
@@ -336,6 +334,7 @@ export default function Landing() {
   const shouldDisableNFTRoutes = useDisableNFTRoutes()
   const originCountry = useAppSelector((state: AppState) => state.user.originCountry)
   const renderUkSpecificText = Boolean(originCountry) && originCountry === 'GB'
+  const [initiallyVisible] = useState(!!originCountry)
```

Comment:
probably better to use a ref here? generally feels like a code smell though that you need to do this gymnastic